File: lesson_5_2/app/src/main/java/com/kalashnyk/denys/samplemvvmapp/repository/database/AppDatabase.java
package com.kalashnyk.denys.samplemvvmapp.repository.database;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.kalashnyk.denys.samplemvvmapp.repository.database.dao.UserDao;
import com.kalashnyk.denys.samplemvvmapp.repository.database.entity.UserEntity;

@Database(entities = {UserEntity.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract UserDao userDao();
}

File: lesson_7/app/src/main/java/com/kalashnyk/denys/kotlinsample/repository/database/AppDatabase.kt
package com.kalashnyk.denys.kotlinsample.repository.database

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase
import com.kalashnyk.denys.kotlinsample.repository.database.dao.UserDao
import com.kalashnyk.denys.kotlinsample.repository.database.entity.UserEntity

@Database(entities = [(UserEntity::class)], version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun userDao(): UserDao
}